camelot/view/controls/editors/textlineeditor.py

```diff
@@ -43,8 +43,10 @@ def __init__(self,
                  echo_mode=None,
                  column_width=None,
                  action_routes=[],
+                 validator_type=None,
                  field_name='text_line'):
         CustomEditor.__init__(self, parent, column_width=column_width)
+        self.validator_type = validator_type
```

Comment:
het is natuurlijk wel zo, dat als we hier overal de validator type gaan moeten zetten als member, we evengoed een validator kunnen constueren ...